spring-cloud-aws-samples/spring-cloud-aws-s3-sample/app-transfermanager/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think it would be better if we just have single sample project for S3. Meaning, include transfer manager in existing sample

Reply:
I agree, the only reason why I added a new sample project was to showcase the autoconfiguration and the S3OutputStream implementation that is used on application startup. Will instead simply create the example as an extra controller on the current sample app and remove the second one